src/Microsoft.DotNet.Darc/Darc/Program.cs

```diff
@@ -38,7 +38,9 @@ private static int Main(string[] args)
             options = GetOptions();
         }
 
-        return Parser.Default.ParseArguments(args, options)
+        Parser parser = new Parser(settings => { settings.AutoVersion = false; settings.HelpWriter = Console.Error; });
```

Comment:
What does this do? Are we still keeping the `darc --version`?

Reply:
Can you add a description in the PR what this change does?

Reply:
the original change didn't do it, but maybe we should keep it. I reverted it back so it only works on this one command